06-hot-reloading-with-directives/services/reviews/schema.graphql

```diff
@@ -0,0 +1,37 @@
+type Review {
+  id: ID!
+  body: String
+  author: User
+  product: Product
+}
+
+type User @key(selectionSet: "{ id }") {
+  id: ID!
+  username: String
+  numberOfReviews: Int
+  reviews: [Review]
+}
+
+type Product @key(selectionSet: "{ upc }") {
```

Comment:
Wasn't `@base` selected as the directive here...? This looks like Federation written incorrectly...

Reply:
I switched base to key as if you are not using computed fields, base kind of means nothing

Reply:
Reluctantly, open to suggestions....

Reply:
You can customize the name, right? If so, then it’s fine. I’m just thinking ahead to the inevitable jerkface that wants to both stitch and federate their schema, and shows up complaining that the SDLs overlap and federation got there first. As long as there’s an easy answer of, “change the name”, that seems moot. I do agree that “key” is better.

Reply:
👍